lti_consumer/plugin/views.py

```diff
@@ -118,7 +118,7 @@ def public_keyset_endpoint(request, usage_id=None):
         response['Content-Disposition'] = 'attachment; filename=keyset.json'
         return response
     except (LtiError, InvalidKeyError, ObjectDoesNotExist) as exc:
-        log.info("Error while retrieving keyset for given usage_id.")
+        log.info("Error while retrieving keyset for usage_id %s: %r", usage_id, exc)
```

Comment:
The `%s` and `%r` are backwards.  We want the repr() for the usage_id where we might be unsure of the type or trailing white space.  The %s is for the exception, to turn it into its message.

Reply:
(sorry, throughout...)

Reply:
Sorry, my mistake. I've fixed the occurrences on the log statements.